src/commands/init.ts

```diff
@@ -0,0 +1,539 @@
+      return platform;
+    }
+
+    if (flagsValue) {
+      return flagsValue;
+    }
+
+    const project = projects.find((project) => {
+      return project.project_id === appName;
+    });
+
+    if (!project) {
+      return 'static';
+    }
+    return project!.type;
+  }
+
+  async getAppPort(
+    platform: string,
+    flagValue: number | undefined,
+    projects: IProject[],
+  ): Promise<number> {
+    if (flagValue) {
+      return flagValue;
+    }
+
+    const defaultPort = getPort(platform);
+
+    if (!defaultPort) {
+      const port = await promptPort(platform);
+      return port;
+    }
+    return defaultPort;
+  }
+
+  async buildLocationPrompt(flagValue: string | undefined): Promise<string> {
+    if (flagValue) {
+      return flagValue;
+    }
+
+    const { location } = (await inquirer.prompt({
+      message: 'Specify the build location: ',
+      name: 'location',
+      type: 'list',
+      default: 'iran',
+      choices: ['iran', 'germany'],
+    })) as { location: string };
+    return location;
+  }
+
+  async promptPlatformVersion(
+    platform: string,
+    flagValue: string | undefined,
+  ): Promise<string | undefined> {
+    if (flagValue) {
+      return flagValue;
+    }
+
+    const versions = supportedVersions(platform);
+
+    if (versions) {
+      let message: string | undefined;
+      if (['flask', 'django'].includes(platform)) {
+        message = 'Select python version';
+      }
+
+      if (platform === 'laravel') {
+        message = 'Select php version';
+      }
+
+      if (platform === 'next') {
+        message = 'Select node version';
+      }
+
+      if (!message) {
+        message = `Selcet ${platform} version: `;
+      }
+
+      const { version } = (await inquirer.prompt({
+        message: message || 'Select platform version',
+        name: 'version',
+        type: 'list',
+        default: versions.defaultVersion,
+        choices: versions.allVersions,
+      })) as { version: string };
+      return version;
+    }
+  }
+
+  async createLiaraJsonFile(configs: ILiaraJSON) {
+    try {
+      this.spinner.start();
+
+      await fs.writeFile(
+        `${process.cwd()}/liara.json`,
+        JSON.stringify(configs, null, 2),
+      );
+      this.spinner.succeed('liara.json is successfully created!');
+    } catch (error) {
+      throw new Error('There was a problem while creating liara.json!');
+    }
+  }
+
+  setLiaraJsonConfigs(
+    port: number,
+    appName: string,
+    buildLocation: string,
+    platform: string,
+    platformVersion: string | undefined,
+    diskConfigs: { disk: string; path: string }[] | undefined,
+    healthCheck?: IHealthConfig | undefined,
+    cron?: string[] | undefined,
+  ): ILiaraJSON {
+    const versionKey = this.setVersionKey(platform, platformVersion);
+
+    const configs: ILiaraJSON = {
+      port,
+      app: appName,
+      build: {
+        location: buildLocation,
+      },
+    };
```

Comment:
Shouldn't we also include the platform in the liara.json file?
Especially when using the -y flag, since the platform is already detected and its version is written to the liara.json file.